pkg/timeline/router.ts

```diff
@@ -96,6 +97,40 @@ export const CreateListRoute = createRoute({
   },
 });
 
+export const FetchListRoute = createRoute({
+  method: 'get',
+  tags: ['timeline'],
+  path: '/lists',
```

Comment:
このようにしないとうまくルーティングできない気がします
```suggestion
  path: '/lists/:id',
```